### PR TITLE
Integrations XML improvement

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-808)

## Description
Used the following library: 
https://pypi.org/project/defusedxml/

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by dansterenson** [comment]
@DeanArbel As we wait for a response, you can go over the PR.

File: Packs/ArcherRSA/ReleaseNotes/1_2_5.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### RSA Archer v2
+- Updated the Docker image to: *demisto/python3:3.10.9.45313*.
+- Changed the XML parsing to be more secure.
+- Replaced xml strings with xml objects.
```
Comment (DeanArbel) [inline Packs/ArcherRSA/ReleaseNotes/1_2_5.md:6]: This is more of a technical change, so no need to add it as part of the RN.
```suggestion
```

File: Packs/SymantecMSS/ReleaseNotes/1_0_5.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Symantec Managed Security Services
+- Updated the Docker image to: *demisto/chromium:1.0.0.45748*.
+- Changed the XML parsing to be more secure.
+- Replaced xml strings with xml objects.
```
Comment (DeanArbel) [inline Packs/SymantecMSS/ReleaseNotes/1_0_5.md:6]:
```suggestion
```

File: Packs/TAXIIServer/ReleaseNotes/2_0_22.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### TAXII Server
+- Changed the XML parsing to be more secure.
+- Replaced xml strings with xml objects.
```
Comment (DeanArbel) [inline Packs/TAXIIServer/ReleaseNotes/2_0_22.md:5]:
```suggestion
```

**Review by DeanArbel — APPROVED** [review]
Well done

**Comment by dansterenson** [comment]
@DeanArbel I updated some docker images. since you last checked it. We need to force merge: coverage, and some test playbook fails that are related to packs that there docker images were updated (seems like credentials issues).

**Comment by dansterenson** [comment]
Force merge needed ([ERROR]: Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.yml: [IN140] - The integration Symantec MSS is currently in skipped. Please add working tests and unskip. Skip comment: No instance, probably not going to get it (issue 15513))

**Pull request merged**